components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -293,7 +249,10 @@ open class GleanInternalAPI {
 
         when (pingEvent) {
             Glean.PingEvent.Background -> {
-                sendPing("baseline", "baseline")
+                if (baselinePingScheduler.canSendPing()) {
+                    sendPing(BaselinePingScheduler.STORE_NAME, BaselinePingScheduler.STORE_NAME)
+                }
```

Comment:
Do we need to call `baselinePingScheduler.updateSentTimestamp()` here?

Reply:
Yes, good catch!  We aren't really doing anything with it at the moment but it may become useful in the future to determine the last time we sent a ping.